src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -96,7 +103,100 @@ private function fixFunction(Tokens $tokens, $startFunctionIndex)
 
             if ($token->equals(',')) {
                 $this->fixSpace2($tokens, $index);
+                if (!$isMultiline && $this->isNewline($tokens[$index + 1])) {
+                    $isMultiline = true;
+                }
+            }
+        }
+
+        return $isMultiline;
+    }
+
+    private function ensureFunctionFullyMultiline(Tokens $tokens, $startFunctionIndex)
+    {
+        $endFunctionIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $startFunctionIndex);
+        // find out what the indentation is
+        $searchIndex = $startFunctionIndex;
+        do {
+            $prevWhitespaceTokenIndex = $tokens->getPrevTokenOfKind(
+                $searchIndex,
+                array(array(T_WHITESPACE))
+            );
+            $searchIndex = $prevWhitespaceTokenIndex;
+        } while (
+            $prevWhitespaceTokenIndex
+            && strpos($tokens[$prevWhitespaceTokenIndex]->getContent(), "\n") === false
+        );
+        $existingIndentation = $prevWhitespaceTokenIndex ?
+            ltrim($tokens[$prevWhitespaceTokenIndex]->getContent(), "\n") :
+            '';
+
+        $indentation = $existingIndentation.'    ';
+        $endFunctionIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $startFunctionIndex);
+        if (!$this->isNewline($tokens[$endFunctionIndex - 1])) {
+            $tokens->insertAt(
+                $endFunctionIndex,
+                new Token(array(T_WHITESPACE, "\n$existingIndentation"))
```

Comment:
This is not valid in case the file uses line endings different than `\n` (e.g. `\r\n`). Please add test cases that cover that.